### DISCUSSION TITLE
有没有可能给扩容盘使用Ventoy?

### DISCUSSION THREAD

**Discussion by txmu** [ventoy/Ventoy#1741] [Q&A]
比如一个虚标的总容量为57.9G，实际容量为29.7G的U盘，有没有可能在其上通过“在磁盘尾部留出一段空间（如29G）”的方法来正常安装并使用Ventoy?（只是单纯探讨，不考虑扩容盘可能容易丢失数据以及这么小的剩余真实可用空间是否有实用价值的问题）

**Comment by ventoy** [comment]
可以的。
只要前面 Ventoy 的2个分区所在的区域保证没有问题就可以正常使用。
